vpktool.py

```diff
@@ -0,0 +1,50 @@
+# Easy VPK extractor
```

Comment:
Need a shebang for python, so it can be executed directly.

`#!/usr/bin/env python`